scripts/assemble_univariate_priors.py

```
@@ -0,0 +1,1314 @@
+            result = minimize(
+                neg_log_likelihood,
+                init_params,
+                method="L-BFGS-B",
+                bounds=bounds,
+                options={"maxiter": 2000, "ftol": 1e-10},
+            )
+            if result.fun < best_nll:
+                best_nll = result.fun
+                best_result = result
+        except Exception:
+            continue
```

Ruff (0.15.15)

[error] `try`-`except`-`continue` detected, consider logging the exception

(S112)